terpnet/build.yml

```
@@ -0,0 +1,25 @@
+version: '3.4'
+
+services:
+  node_1:
+    build:
+      context: ../
+      args:
+        PROJECT: terpnet
+        VERSION: v0.1.2
+        REPOSITORY: https://github.com/terpnetwork/terpnetwork
```

Should this be `https://github.com/terpnetwork/terp-core` ?

Omnibus also needs to know the binary and directory. The namespace is (almost?) always the binary name in uppercase which is the default, but I usually include it so it's clear.

```suggestion
        REPOSITORY: https://github.com/terpnetwork/terp-core
        PROJECT_BIN: terpd
        PROJECT_DIR: .terp
        NAMESPACE: TERPD
```